dist/bundles/thunk_bundle.js

```diff
@@ -7,10 +7,17 @@ module.exports = (() => {
   var __getOwnPropNames = Object.getOwnPropertyNames;
   var __getProtoOf = Object.getPrototypeOf;
   var __hasOwnProp = Object.prototype.hasOwnProperty;
+  var __require = /* @__PURE__ */ ((x) => typeof require !== "undefined" ? require : typeof Proxy !== "undefined" ? new Proxy(x, {
+    get: (a, b) => (typeof require !== "undefined" ? require : a)[b]
+  }) : x)(function(x) {
+    if (typeof require !== "undefined")
+      return require.apply(this, arguments);
+    throw new Error('Dynamic require of "' + x + '" is not supported');
```

Comment:
Hmm. I'm trying to figure if if there's a way I can use`v8.serialize` in SDK code that doesn't end up triggering some kind of `Dynamic require of ... is not supported` error

Reply:
is v8 not available in isolated-vm?

Reply:
oh so isolated-vm doesn't have v8 in its runtime. interesting even if it mentions v8 thousands times in its readme..

```
> context.evalSync('v8.serialize')
Uncaught ReferenceError: v8 is not defined
    at <isolated-vm>:1:1
    at (<isolated-vm boundary>)
    at REPL8:1:9
```

Reply:
it only has language primitives. I guess you don't want to compile a full v8 module into isolated-vm issue. maybe the only option is to inject a function that calls back to the native v8 module.